Tools/TestRunnerShared/CMakeLists.txt

```diff
@@ -48,19 +48,14 @@ GENERATE_BINDINGS(TestRunnerSharedBindings
     DESTINATION ${TestRunnerShared_DERIVED_SOURCES_DIR}
     GENERATOR DumpRenderTree)
 
-set(TestRunnerShared_LIBRARY_TYPE OBJECT)
 set(TestRunnerShared_DEPENDENCIES TestRunnerSharedBindings)
-
-set(TestRunnerShared_INTERFACE_LIBRARIES
-    $<TARGET_OBJECTS:TestRunnerShared>
-    TestRunnerShared
-)
+set(TestRunnerShared_INTERFACE_LIBRARIES TestRunnerShared)
```

Comment:
Why does `TestRunnerShared` have to have `TestRunnerShared` itself as a interface librairy.

Reply:
It’s for the WebKit::TestRunnerShared target.

Reply:
It's not clear to me. It'd be better if `WEBKIT_LIBRARY` can become easier to use.

Reply:
I can make it so it automagically adds the target to `${_target}_INTERFACE_LIBRARIES`.